From 8d8519928c2475f2454e33f47437850761f444a8 Mon Sep 17 00:00:00 2001
From: Brandon Smith <brandon.smith@drs.com>
Date: Tue, 11 Apr 2017 09:35:58 -0400
Subject: [PATCH] [PATCH] Fix pthread compile for Picoflexor OE

---
 gnuradio-runtime/CMakeLists.txt |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/gnuradio-runtime/CMakeLists.txt b/gnuradio-runtime/CMakeLists.txt
index 7660642..ad9aec9 100644
--- a/gnuradio-runtime/CMakeLists.txt
+++ b/gnuradio-runtime/CMakeLists.txt
@@ -141,6 +141,14 @@ install(FILES
 )
 endif(ENABLE_GR_LOG AND HAVE_LOG4CPP)
 
+find_package(Threads REQUIRED)
+if(THREADS_HAVE_PTHREAD_ARG)
+    target_compile_options(PUBLIC gnuradio-runtime -pthread)
+endif()
+if(CMAKE_THREAD_LIBS_INIT)
+    target_link_libraries(gnuradio-runtime "${CMAKE_THREAD_LIBS_INIT}")
+endif()
+
 ########################################################################
 # Add subdirectories
 ########################################################################
-- 
1.7.9.5

